frontend/Store.js

```diff
@@ -150,6 +151,7 @@ class Store extends EventEmitter {
     this._establishConnection();
     this._eventQueue = [];
     this._eventTimer = null;
+    this._emitSearchRootsChange = debounce(this.filterSearchRoots, 100);
```

Comment:
@Timothee Thanks for the heads up. I actually needed to add the type check function with `Function` in `Store`.